kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -1448,6 +1455,22 @@ showDotGraph gr =
         . Graph.graphToDot (graphParams gr)
         $ gr
 
+-- | A version of @showDotGraph@ that catches any exceptions.
+showDotGraphCatchException ::
+    From axiom AttrLabel.Label =>
+    From axiom RuleIndex =>
+    Gr CommonClaimState (Maybe (Seq axiom)) ->
+    IO ()
+showDotGraphCatchException gr =
+    catch (showDotGraph gr) $ \(e :: SomeException) ->
+        hPrint stderr $
+            Pretty.vsep
+                [ "Encountered the following exception:"
+                , Pretty.indent 4 $ fromString $ displayException e
+                , "Please note that the graph command is not\
+                  \ currently supported on MacOS."
```

Comment:
Let's recommend using `graph` with a file name argument, which works on MacOS as well. This saves the graph image to the file.